### PR TITLE
FALCON-1895 Refactoring of FalconCLI and FalconClient

### PR DESCRIPTION
**Pull request by PraveenAdlakha**

### CODE REVIEW

**Comment by bvellanki** [comment]
+1 on moving String constants to FalconCLIConstants.

File: client/pom.xml [hunk 1]
```diff
@@ -111,6 +111,7 @@
             <artifactId>hive-webhcat-java-client</artifactId>
             <version>${hive.version}</version>
         </dependency>
+
```
Comment (sowmyaramesh) [inline client/pom.xml]: minor nit: Remove extra line

Comment (sowmyaramesh) [inline client/pom.xml]: +1, looks good!

**Comment by peeyushb** [comment]
@PraveenAdlakha Patch looks good. +1.

**Comment by PraveenAdlakha** [comment]
Thanks for the review everyone..

**Comment by peeyushb** [comment]
If there are no any further comments, I will commit the patch soon.

**Pull request closed without merging**